test/unit/query-generator/show-tables.query.test.ts

```diff
@@ -0,0 +1,55 @@
+import {
+  expectsql,
+  createSequelizeInstance,
+} from '../../support';
+
+const sequelize = createSequelizeInstance();
+const queryGenerator = sequelize.getQueryInterface().queryGenerator;
+
+const sequelizeWithSchema = createSequelizeInstance({ schema: 'schema_at_init' });
+const queryGeneratorWithSchema = sequelizeWithSchema.getQueryInterface().queryGenerator;
+
+describe('QueryGenerator#showTablesQuery', () => {
+
+  it('should use the schema from initialization options', async () => {
+    expectsql(queryGeneratorWithSchema.showTablesQuery(), {
```

Comment:
Can you declare `showTablesQuery` on `abstract/query-generator.d.ts`? It's needed to fix this typing issue

Reply:
Addressed this in:
1. https://github.com/sequelize/sequelize/pull/15101/commits/86304e3ba675ea715383cff42773a221b7207124
2. Added an overload for Snowflake - https://github.com/sequelize/sequelize/pull/15101/commits/d528b54136dfb3b9e8742bb8bfae48528eddb516

https://github.com/sequelize/sequelize/pull/15101/commits/86304e3ba675ea715383cff42773a221b7207124